### ISSUE TITLE
Cannot enable Auto-start

### ISSUE THREAD

**Issue by sorry-i-am-late** [azymohliad/watchmate#68]
When I attempt to enable Auto-start in background at login I get an error/popup that says "Not allowed to change autostart setting"

I am using the flatpak on my pinephone running Arch.

**Comment by sorry-i-am-late** [comment]
Here is my work around for this issue as well as the auto reconnect imperfections.

[https://github.com/azymohliad/watchmate/pull/61#issuecomment-2323510023](url)

**Comment by sorry-i-am-late** [comment]
I am using PHOSH if that is what you are asking.

**Comment by azymohliad** [comment]
Yeah, that's what I was asking, thanks. At first glance this is an issue with Phosh implementation of Background portal, but I might be missing something. I'll need to set some time to debug it to have a better answer.
